epub33/core/index.html

```diff
@@ -5529,6 +5529,11 @@ <h4>Path and File Names</h4>
 						</li>
 					</ul>
 
+					<div class="note">
+						<p>EPUB Creators may have to truncate File Names to keep them within the 255 bytes range. When doing so it is important to avoid mid-character truncation due to the difference between bytes and characters in multibyte encodings such as UTF-8. See the section on <a data-cite="international-specs/#char_truncation">"Truncating or limiting the length of strings"</a> in [[international-specs]] for more information. </p>
```

Comment:
Nit: I would probably say "...within the 255 byte limit" rather than "range"

Reply:
Right, will change this